### DISCUSSION TITLE
Help us translate Gluco Check into Swedish

### DISCUSSION THREAD

**Discussion by nielsmaerten** [nielsmaerten/gluco-check#149] [General]
We're working on making Gluco Check available in Swedish asap. But we still need some help:
* [ ] [proofreading the translation](https://translate.glucocheck.app)
* [ ] testing the Swedish version on an actual Google Home device
* [ ] reporting any bugs

If you speak both Swedish and English, and want to help us:
Please leave a comment below to get in touch 😊

Thank you! 👏

PS: If you prefer email, you can reach us at: translations@glucocheck.app

**Comment by DanielDanielsson** [comment]
Hi! 
Im a T1D webdeveloper based in Gothenburg Sweden and I would love to help! I use my google home device every day. 
email: danielssondaniel94@gmail.com

**Comment by C0mplexed** [comment]
Hi! I want to help with translation also. I am a programmer (python, c++, java, html5) and also have top grades in both Swedish and English.

**Reply by nielsmaerten** [reply]
Hey @C0mplexed, 
I saw you already suggested a bunch of improvements to the translation on Crowdin. Awesome! 👍

You've probably noticed this, but the initial Swedish translation was done by Google Translate. So there's likely a number of errors in there.
Would you be willing to go through all of the strings and fix the mistakes?

Once you think the translation is good enough, let me know and I'll get started on deploying it.
Thanks for your help!

PS: I've given you Proofreader permissions on Crowdin: You can now approve translations and reject the bad ones.

**Comment by C0mplexed** [comment]
Of course. I will do that later this week.Thanks!Best Regards - LinusMvh - Linus Forsman
